src/Aspire.Hosting.PostgreSQL/PostgresServerResource.cs

```diff
@@ -17,7 +17,7 @@ public class PostgresServerResource : ContainerResource, IResourceWithConnection
     /// <param name="name">The name of the resource.</param>
     /// <param name="userName">A parameter that contains the PostgreSQL server user name, or <see langword="null"/> to use a default value.</param>
     /// <param name="password">A parameter that contains the PostgreSQL server password.</param>
-    public PostgresServerResource(string name, ParameterResource? userName, ParameterResource password) : base(name)
+    public PostgresServerResource(string name, ParameterResource userName, ParameterResource password) : base(name)
```

Comment:
We should be consistent across all resources that take a user name.

https://github.com/dotnet/aspire/blob/9fab2f6bd66866fb210f3b6c1f6347a73c9ab57a/src/Aspire.Hosting.RabbitMQ/RabbitMQServerResource.cs#L21